editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx

```diff
@@ -945,7 +985,7 @@ describe('conditionals in the navigator', () => {
       await dispatchDone
     })
 
-    act(() =>
+    await act(() =>
```

Comment:
does this need to be wrapped in an act at all? other places dragElement is not wrapped